clos.md

```diff
@@ -406,7 +406,7 @@ The [closer-mop](https://github.com/pcostanza/closer-mop) library is *the*
 portable way to do CLOS/MOP operations.
 
 
-A subclass inherits all of its parents slots, and it can override any
```

Comment:
parents'

Reply:
Thanks for catching that. I pushed again with your fix.